Upgrade: Migrate `max-w-screen-*` candidates

philipp-spiess opened the pull request:
This PR migrates `max-w-screen-*` candidates to the v4 equivalent relying on the breakpoint var: `max-w-[var(--breakpoint-*)]`

philipp-spiess commented:
* **#14754** <a href="https://app.graphite.dev/github/pr/tailwindlabs/tailwindcss/14754?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `next`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @philipp-spiess and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

thecrypticace commented on packages/@tailwindcss-upgrade/src/template/migrate.ts:
```diff
@@ -7,6 +7,7 @@ import { arbitraryValueToBareValue } from './codemods/arbitrary-value-to-bare-va
 import { automaticVarInjection } from './codemods/automatic-var-injection'
 import { bgGradient } from './codemods/bg-gradient'
 import { important } from './codemods/important'
+import { maxWScreen } from './codemods/max-w-screen'
```
Can we call this maxWidthScreen instead?

philipp-spiess commented:
### Merge activity

* **Oct 22, 12:07 PM EDT**: The merge label 'merge-queue' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=tailwindlabs&repo=tailwindcss) once it meets the requirements.

philipp-spiess committed:
Upgrade: Migrate `max-w-screen-*` candidates (#14754)

This PR migrates `max-w-screen-*` candidates to the v4 equivalent relying on the breakpoint var: `max-w-[var(--breakpoint-*)]`
